packages/vm/src/static_analysis.rs

```diff
@@ -41,6 +41,8 @@ pub enum Entrypoint {
     Ibc2PacketReceive,
     #[strum(serialize = "ibc2_packet_timeout")]
     Ibc2PacketTimeout,
+    #[strum(serialize = "ibc2_packet_send")]
+    Ibc2PacketSend,
```

Comment:
Oh, looks like we forgot to add the ack entrypoint here. Not related to this PR though.